lib/src/json_schema/json_schema.dart

```diff
@@ -1162,8 +1174,8 @@ class JsonSchema {
 
   /// The value of the exclusiveMaximum for the [JsonSchema], if any exists.
   num get exclusiveMaximum {
-    // If we're on draft6, the property contains the value, return it.
-    if ([SchemaVersion.draft6, SchemaVersion.draft7].contains(schemaVersion)) {
+    // If we're beyond draft4, the property contains the value, return it.
+    if (schemaVersion != SchemaVersion.draft4) {
```

Comment:
`>` would probably be clearer than `!=`

Reply:
Maybe we should change this to `schemaVersion >= SchemaVersion.draft6` for consistency.